src/codemagic/tools/_app_store_connect/action_groups/apps_action_group.py

```diff
@@ -136,3 +156,121 @@ def list_app_store_versions_for_app(
             app_store_versions_filter,
             should_print,
         )
+
+    @cli.action(
+        'expire-builds',
+        AppArgument.APPLICATION_ID_RESOURCE_ID,
+        BuildArgument.BUILD_ID_RESOURCE_ID_EXCLUDE_OPTIONAL,
+        action_group=AppStoreConnectActionGroup.APPS,
+    )
+    def expire_builds(
+        self,
+        application_id: ResourceId,
+        excluded_build_id: Optional[Union[ResourceId, Sequence[ResourceId]]] = None,
+        should_print: bool = False,
+    ) -> List[Build]:
+        """
+        Expire all application builds except the given build(s)
+        """
+        builds_to_skip: Set[ResourceId] = set()
+
+        if isinstance(excluded_build_id, ResourceId):
+            builds_to_skip.add(excluded_build_id)
+        elif excluded_build_id is not None:
+            builds_to_skip.update(excluded_build_id)
+
+        builds = self.list_builds(application_id=application_id, not_expired=True, should_print=should_print)
+        return [
+            self.expire_build(
+                build_id=build.id,
+            )
+            for build in builds
+            if build.id not in builds_to_skip
+        ]
+
+    @cli.action(
+        'expire-build-submitted-for-review',
+        AppArgument.APPLICATION_ID_RESOURCE_ID,
+        action_group=AppStoreConnectActionGroup.BUILDS,
+    )
+    def expire_build_submitted_for_review(
+        self,
+        application_id: ResourceId,
+        should_print: bool = False,
+    ) -> Optional[Build]:
+        """
+        Expire application build that is currently waiting for review, or is currently in review
```

Comment:
This applies only for TestFlight builds, right? Maybe it would be better to somehow reflect that in the documentation too then?